ocis/go.mod

```diff
@@ -34,6 +34,7 @@ require (
 )
 
 replace (
+	github.com/glauth/glauth => ../../glauth
```

Comment:
wrong replace path. We need the monorepo relative path `../glauth`

Reply:
wtf